source/_integrations/starlink.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Starlink
+description: Instructions on how to integrate Starlink into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2022.9
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@boswelja'
+ha_domain: starlink
+ha_dhcp: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Starlink integration allows you to integrate your [Starlink](https://www.starlink.com/) into Home Assistant.
+
+{% include integrations/config_flow.md %}
```

Comment:
Does the integration work in bypass mode? If not we should tell them it doesn't

Reply:
I'm not sure what bypass mode is. If there's extra steps needed to support it, then it doesn't support it

Reply:
https://support.starlink.com/
> Standard(Rectangular) Starlink
>
> Bypass mode will completely disable the built-in Starlink WiFi router functionality. An Ethernet Adapter is required to > use this feature.
> 
> Factory reset the Starlink WiFi router to disable bypass mode.

https://community.ui.com/questions/Starlink-Square-second-generation-how-to-connect-to-USG/6300aec1-3e9f-46c1-a808-4d913fe8107a


I have mine setup in bypass mode.  I'll have to factory reset it this weekend to do testing.

Reply:
Oh bypass mode on dishy, I thought you were referring to some homeassistant feature 😅
As far as I'm aware, bypass mode shuts down the local API, though this may have changed. I will update the documentation

Reply:
I added a quick sentence on bypass mode